# Path2DWithAngles.msg

PointAndAngle[] path